ddtrace/internal/datadog/profiling/dd_wrapper/src/uploader_builder.cpp

```diff
@@ -136,8 +136,11 @@ Datadog::UploaderBuilder::build()
     }
 
     // If we're here, the tags are good, so we can initialize the exporter
-    ddog_prof_Exporter_NewResult res = ddog_prof_Exporter_new(
-      to_slice("dd-trace-py"), to_slice(profiler_version), to_slice(family), &tags, ddog_Endpoint_agent(to_slice(url)));
+    ddog_prof_Exporter_NewResult res = ddog_prof_Exporter_new(to_slice("dd-trace-py"),
+                                                              to_slice(profiler_version),
+                                                              to_slice(family),
+                                                              &tags,
+                                                              ddog_prof_Endpoint_agent(to_slice(url)));
```

Comment:
format?

Reply:
This is how our c++ style rules want to format the section, I think.  At least it appears to pass in CI.

If you're seeing something different on your local machine, it should be noted that clang-format linter appears to do  slightly different things depending on the version of clang.  I'm not 100% sure what CI uses.  Maybe 13?